src/App.jsx

```diff
@@ -1,107 +1,20 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+const preparingPosts = postsFromServer.map(post => ({
+  ...post,
+  user: usersFromServer.find(user => (user.id === post.userId)),
+  comments: commentsFromServer.filter(comment => (comment.postId === post.id)),
+}));
```

Comment:
Would be better to split these with function
```suggestion
const preparingPosts = postsFromServer.map(post => ({
  ...post,
  user: getUserById(...),
  comments: getCommentsById(...),
}));
```